src/request/ErrorResponse.ts

```diff
@@ -6,12 +6,19 @@ interface ErrorBody {
   additionalErrorInfo?: AdditionalErrorInfo;
 }
 
-interface AdditionalErrorInfo {
+export interface AdditionalErrorInfo {
```

Comment:
Could we maybe call this PartitionedQueryExecutionInfo or something along those lines? I know that this is the property name we use in the error body, but it is not very descriptive (as it is in the error body itself). We'll end up using this same object body once Query Plan becomes available soon-ish.